test/index.test.js

```diff
@@ -121,6 +130,12 @@ describe('Protractor Flake', () => {
   })
 
   context('options', () => {
+    it('allows a different path for protractor by using protractorPath option', () => {
+      protractorFlake({protractorPath: pathToProtractor()})
```

Comment:
I think i'd prefer hardcoding this to an arbitrary path like `/usr/local/protractor` (it doesn't need to actually resolve since we are stubbing out `spawn`).